InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ball_management/BallManagementSubsystem.java

```diff
@@ -8,6 +8,7 @@
 import frc.robot.config.Config;
 import frc.robot.subsystem.BitBucketSubsystem;
 import frc.robot.utils.talonutils.MotorUtils;
+import io.github.oblarg.oblog.annotations.Log;
 
 public class BallManagementSubsystem extends BitBucketSubsystem {
```

Comment:
Since the BMS is basically part of the shooter, can you override the layoutName for oblog and make it put components in the Shooter tab?